.github/workflows/try.yml

```
@@ -20,13 +24,15 @@ jobs:
     steps:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
       - uses: actions/checkout@v2
-      #- uses: actions/checkout@master
-      #  with:
-      #    name: qiboteam/qibo 
-      - run: git pull
-      - run: git clone https://github.com/qiboteam/qibo.git  
-      - run: ls -l
+
+      - name: Clone qibo repository
+        uses: actions/checkout@v2
+        with:
+          repository: qiboteam/qibo
+          path: "~/"
+      - run: git clone https://github.com/qiboteam/qibo.git
       - name: Commit & Push changes
-        uses: actions-js/push@master
+        uses: actions-js/push@work2
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
+      - run: ls -a
```

I guess this you needed for debugging. Add a comment before telling so:
```suggestion
      # TODO: debug, remember to drop this
      - run: ls -a
```